tests/unit/ai/review/test_checklist_selector.py

```
@@ -0,0 +1,290 @@
+
+def test_dual_axis_item_requires_both_domain_and_language() -> None:
+    """Items with both axes only match when domain and language intersect."""
+    custom_item = ChecklistItem(
+        id=CUSTOM_CHECKLIST_ID_START,
+        question="Does any API handler skip auth?",
+        domains=(FileDomain.API,),
+        languages=("python",),
+        category=ReviewCategory.SECURITY,
+        tier=2,
+    )
+    items = list(BUILTIN_CHECKLIST_ITEMS) + [custom_item]
```

Ruff (0.15.20)

[warning] Consider `[*list(BUILTIN_CHECKLIST_ITEMS), custom_item]` instead of concatenation

Replace with `[*list(BUILTIN_CHECKLIST_ITEMS), custom_item]`

(RUF005)